lib/chainsync/chainsync/postgres_config.py

```diff
@@ -0,0 +1,70 @@
+"""Defines the postgres configuration from env vars."""
+
+import os
+from dataclasses import dataclass
+
+from dotenv import load_dotenv
+
+
+@dataclass
+class PostgresConfig:
+    """The configuration dataclass for postgres connections.
+
+    Replace the user, password, and db_name with the credentials of your setup.
+
+    Attributes
+    ----------
+    POSTGRES_USER : str
+        The username to authenticate with
+    POSTGRES_PASSWORD : str
+        The password to authenticate with
+    POSTGRES_DB : str
+        The name of the database
+    POSTGRES_HOST : str
+        The hostname to connect to
+    POSTGRES_PORT : int
+        The port to connect to
+    """
+
+    # default values for local postgres
+    # Matching environment variables to search for
+    # pylint: disable=invalid-name
+    POSTGRES_USER: str = "admin"
+    POSTGRES_PASSWORD: str = "password"
+    POSTGRES_DB: str = "postgres_db"
+    POSTGRES_HOST: str = "localhost"
+    POSTGRES_PORT: int = 5432
```

Comment:
do you think we should add the function to write the class as JSON for all of these config dataclasses?

Reply:
Unsure what this function would be used for, only thing I see it being useful for is to print out a default env file based on this file. Currently, there's no script that will do this for you, and templates are being handled via env.sample files. Let's push this down the road until we see a use case.